ui/src/api.ts

```diff
@@ -25,14 +25,25 @@ export async function getAllPapers() {
 }
 
 export async function papers(s2Ids: string[]) {
-  const data = await doGet(
-    axios.get("/api/v0/papers", {
-      params: {
-        id: s2Ids.join(",")
-      }
-    })
-  );
-  return (data || []) as Paper[];
+  const PAPER_REQUEST_BATCH_SIZE = 50;
+  var paperChunk = [];
```

Comment:
nit pick: how about this name, which conveys that the type of data is a group of S2 IDs, and uses the consistent terminology "batch"?

```suggestion
  var s2IdsBatch = [];
```